apps/aecore/lib/aecore/chain/chain_state.ex

```diff
@@ -49,8 +49,8 @@ defmodule Aecore.Chain.ChainState do
   Builds a merkle tree from the passed chain state and
   returns the root hash of the tree.
   """
-  @spec calculate_txs_hash(chainstate()) :: binary()
-  def calculate_txs_hash(chainstate) do
+  @spec calculate_root_hash(chainstate()) :: binary()
```

Comment:
this function does `calculate_root_hash`

Reply:
the root_hash (or (chain) state root hash) is the merkle root hash of the state (tree) at the height of this block